assets/src/components/activities/image_hotspot/ImageHotspotDelivery.tsx

```diff
@@ -110,7 +111,7 @@ const ImageHotspotComponent: React.FC = () => {
     }
   };
 
-  const mapName = 'map' + model.choices[0].id;
+  const mapName = 'map' + guid();
```

Comment:
With this approach, the `mapName` changes on every render.  That might pose a problem. Instead, you should capture the result of `guid()` in state or via `useRef`

Reply:
That's why I avoided generating a guid the first time around! Not clear to me this causes any problem, though.

Reply:
Fixed